packages/pages/src/nft/[id]/page.tsx

```
@@ -99,13 +99,13 @@ export default async function NFT(props: { params: Promise<{ id: string }> }) {
                 <label className="inline-block w-32 text-slate-400 font-semibold">
                   Chain:
                 </label>{' '}
-                <span>{nft.coinLabel}</span>
+                <span>{nft.chainName}</span>
               </p>
               <p className="mt-4">
                 <label className="inline-block w-32 text-slate-400 font-semibold">
                   Network:
                 </label>{' '}
```

Biome (1.9.4)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)